next.config.js

```diff
@@ -1,6 +1,11 @@
+// This file sets a custom webpack configuration to use your Next.js app
+// with Sentry.
+// https://nextjs.org/docs/api-reference/next.config.js/introduction
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
```

Comment:
Do we always need to add comments like this when using 3rd party tools?

Reply:
I believe those comments are auto generated from the sentry setup tool.